config/routes.rb

```diff
@@ -5,6 +5,10 @@
   # root "articles#index"
```

Comment:
- [ ] Let's remove the commented code. This makes it harder to read the code.

Reply:
@arslanbisharat Thank you for the feedback I will make the changes right away